control/tests/test_hotstore.py

```diff
@@ -29,7 +28,8 @@ def setUp(self):
 
         self.H1 = Hotstore(
             video_object=VP,
-            upload_filepath=self.upload_filepath
+            upload_filepath=self.upload_filepath,
+            video_proto=VP
```

Comment:
it's expecting this as arg, if you can see the first parameter that's the video_proto.